.vscode/settings.json

```diff
@@ -17,6 +17,8 @@
     "eslint.format.enable": true,
     "javascript.format.insertSpaceAfterOpeningAndBeforeClosingNonemptyBraces": false,
     "typescript.format.insertSpaceAfterOpeningAndBeforeClosingNonemptyBraces": false,
+    "javascript.format.enable": false,
```

Comment:
The files are still formatted, just by eslint instead of vscode's formatter. Previously they would both run and conflict and cause all sorts of issues.